### PR TITLE
Add collision meshes. Scale dae meshes consistently.

### PR DESCRIPTION
**Pull request by artnie**
The collision meshes are a copy of the visual meshes. Some dae meshe were scaled to its correct size, but the cram_bullet_world can not use the scaling data from the meshes jet. Therefore the meshes now have a consistent scaling factor of 1.0 and the dimensions take care of the size.

### CODE REVIEW

File: iai_kitchen/urdf/sink_area.urdf.xacro [hunk 1]
```diff
@@ -19,6 +19,12 @@
           <box size=".1 .1 .1" />
         </geometry>
       </visual>
+      <collision>
```
Comment (gaya-) [inline iai_kitchen/urdf/sink_area.urdf.xacro]: @artnie Please get rid of this collision object. It's a virtual link, we don't need a collision object for it.

File: iai_kitchen/urdf/IAI_kitchen.urdf.xacro [hunk 2]
```diff
@@ -40,7 +40,7 @@
   <xacro:include filename="$(find iai_kitchen)/urdf/IAI_sink_area.urdf.xacro"/>
   <xacro:include filename="$(find iai_kitchen)/urdf/IAI_fridge_area.urdf.xacro"/>
   <xacro:include filename="$(find iai_kitchen)/urdf/IAI_tables.urdf.xacro"/>
-  <xacro:include filename="$(find iai_kitchen)/urdf/IAI_kitchen_walls.urdf.xacro"/>
+  <!--xacro:include filename="$(find iai_kitchen)/urdf/IAI_kitchen_walls.urdf.xacro"/-->
```
Comment (gaya-) [inline iai_kitchen/urdf/IAI_kitchen.urdf.xacro]: Let's bring back the walls. They can always be commented out in the future.

Comment (gaya-) [inline iai_kitchen/urdf/IAI_kitchen.urdf.xacro]: @artnie

**Comment by gaya-** [comment]
Thanks!

**Comment by daniel86** [comment]
@artnie you seem to have removed some visual information from the collada files. This causes visual artifacts in openEASE. From quick look at the diff these are library_effects and material statements. Please put them in again, and also check if something else was lost.

**Comment by gaya-** [comment]
@daniel86 you can revert to the old version for now in openease. But i cannot update the semantic map on the robot as it will break CRAM. Sorry that the changes caused problem, we should look into fixing remaining issues as soon as possible after the demo.

**Pull request merged**